### DISCUSSION TITLE
GQL client configurations for server's end

### DISCUSSION THREAD

**Discussion by AminRafaey** [kinngh/shopify-node-mongodb-next-app#55] [General]
Hi @kinngh, configuration for gql clients is pretty much great and works amazingly inside a custom app. But for webhook, we can't use the same configured gql client due to different environments. Did you already have done something for it if yes then what is it? If not, then can you please give any suggestions about how to do it?

**Comment by kinngh** [comment]
Use the `clientProvider.offline.graphqlClient({shop: shop})`.

**Reply by AminRafaey** [reply]
Thanks, it worked!
